samples/Samples.Server/Program.cs

```diff
@@ -1,5 +1,6 @@
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
```

Comment:
Maybe take `using Microsoft.AspNetCore.Hosting;` and `using Microsoft.Extensions.Hosting;` under #if / #else too?

Reply:
Yea I wasn't sure what was best practice with that. Will #if/#else them to make it clearer.